Sources/MapboxNavigation/MapView.swift

```diff
@@ -72,17 +72,17 @@ extension MapView {
     func setShowsTileSet(_ isVisible: Bool, withIdentifier tileSetIdentifier: String, layerIdentifier: String) {
```

Comment:
Is it possible to test this method without changes to the SDK?

If not, can you share you vision how would you test this fix if you had to?

Reply:
I think that testing flow is straightforward in this case: create `MapView` instance, wait till style loads up, disable traffic by setting `MapView.showsTraffic = false`, verify that traffic was really disabled by checking if `MapView.showsTraffic` is set to `false`. 

On the other hand major nuance in implementation of this test is `accessToken` handling, since we do not expose tokens `MapView` will not be able to work with mocked one.

Manual test is pretty fast: start active guidance and make sure that `MapView` doesn't render congestions.